src/source.css

```diff
@@ -5589,9 +5591,15 @@ ul.bda-slist {
 }
 
 /* Message hover toolbar */
-.message-2qnXI6.selected-2P5D_Z .buttons-cl5qTG,
-.mouse-mode .message-2qnXI6:hover .buttons-cl5qTG {
-  visibility: hidden !important;
+.wrapper-2aW0bm {
+  background: rgba(0, 0, 0, 0.4) !important;
+  border: 1px solid var(--hover-color) !important;
+}
+.button-1ZiXG9 {
+  color: var(--hover-bright-color) !important;
+}
```

Comment:
Let's also override the default hover color
```css
.button-1ZiXG9:hover {
    background-color: var(--hover-color);
}
```

Reply:
Yeah, had something in mind for the hover too, just forgot to add it lol